WORKSPACE

```diff
@@ -18,7 +18,7 @@ http_file(
     name = "detekt_cli_jar",
     sha256 = "e9710fb9260c0824b3a9ae7d8326294ab7a01af68cfa510cab66de964da80862",
     urls = [
-        # TODO: Add more mirrors, JCenter/Bintray is notoriously bad with 5xx and all sorts of timeouts.
+        "https://search.maven.org/remotecontent?filepath=io/gitlab/arturbosch/detekt/detekt-cli/1.2.0/detekt-cli-1.2.0-all.jar",
```

Comment:
This is the URL the Maven Central website gives. I can change it to the direct artifact one if you wish.

```
https://repo.maven.apache.org/maven2/io/gitlab/arturbosch/detekt/detekt-cli/1.2.0/detekt-cli-1.2.0-all.jar
```